Tests/CommandsTests/BuildCommandTests.swift

```diff
@@ -874,6 +968,17 @@ struct BuildCommandTestCases {
 
     @Test(
         .disabled("Disabled for now because it is hitting 'IR generation failure: Cannot read legacy layout file' in CI (rdar://88828632)"),
+        .tags(
+            .Feature.CommandLineArguments.BuildSystem,
+            .Feature.CommandLineArguments.Configuration,
+        ),
+        .tags(
+            .Feature.CommandLineArguments.VeryVerbose,
```

Comment:
These tags feel very fine-grained and hard to individually maintain, should we use something more general?

Reply:
These tags are currently being used to determine command line feature parity between SwiftBuild and native build system.